### PR TITLE
Remove unneeded characters form `aria-label` and `title` on BreadCrumbs

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
* Part of https://github.com/nextcloud/server/issues/41895

## Summary

Remove unneeded characters like `&quot` form `aria-label` and `title` on BreadCrumbs

🏚️ Before | 🏡 After
---|---
Screenshot from 2023-12-05 14-46-13 | Screenshot from 2023-12-05 14-43-17

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by susnux — APPROVED** [review]

File: apps/files/src/components/BreadCrumbs.vue [hunk 1]
```diff
@@ -110,7 +110,7 @@ export default Vue.extend({
 			if (section?.to?.query?.dir === this.$route.query.dir) {
 				return t('files', 'Reload current directory')
 			}
-			return t('files', 'Go to the "{dir}" directory', section)
+			return t('files', 'Go to the {dir} directory', section)
```
Comment (ShGKme) [inline apps/files/src/components/BreadCrumbs.vue:113]: During the feature and string freeze, we should not create strings for translators.

**Review by ShGKme — CHANGES_REQUESTED** [review]
It seems to me that the issue was only about `aria-label` on the `<li>` element. I check and screen-readers work fine with the escaped HTML quote in the attribute value.

**Comment by JuliaKirschenheuter** [comment]
not relevant, closing PR

**Pull request closed without merging**